mithril-client-cli/src/main.rs

```diff
@@ -43,6 +43,10 @@ pub struct Args {
     /// Override configuration Aggregator endpoint URL.
     #[clap(long, env = "AGGREGATOR_ENDPOINT")]
     aggregator_endpoint: Option<String>,
+
+    /// Draw output logs in JSON format.
```

Comment:
Maybe we should say about that effect of this option which is related to the level of verbosity?
```suggestion
    /// Enable JSON output for logs.
```